### PR TITLE
[MSP] Use MSP2_COMMON[_SET]_SERIAL_CONFIG for configuring ports 

### PR DESCRIPTION
**Pull request by fiam**
Configurator side of https://github.com/betaflight/betaflight/pull/9332

### CODE REVIEW

**Comment by fiam** [comment]
@McGiverGim Sure, I'm on it :-)

**Comment by McGiverGim** [comment]
@fiam thanks for the effort fixing all the Sonar issues!!! Great job!

File: src/js/msp.js [hunk 1]
```diff
@@ -27,112 +60,166 @@ var MSP = {
...
+    _initialize_read_buffer: function() {
+        this.message_buffer = new ArrayBuffer(this.message_length_expected);
+        this.message_buffer_uint8_view = new Uint8Array(this.message_buffer);
+    },
+    _dispatch_message: function(expectedChecksum) {
+        if (this.message_checksum === expectedChecksum) {
+            // message received, store dataview
+            this.dataView = new DataView(this.message_buffer, 0, this.message_length_expected);
+        } else {
+            console.log('code: ' + this.code + ' - crc failed');
```
Comment (McGiverGim) [inline src/js/msp.js]: Strange that Sonar didn't detect this string concatenation...

File: src/js/msp.js [hunk 2]
```diff
@@ -147,13 +234,26 @@ var MSP = {
     clearListeners: function() {
         this.listeners = [];  
     },
-    send_message: function (code, data, callback_sent, callback_msp, doCallbackOnError) {
-        if (code === undefined) {
-            return;
+    crc8_dvb_s2: function(crc, ch) {
+        crc ^= ch;
+        for (let ii = 0; ii < 8; ++ii) {
```
Comment (McGiverGim) [inline src/js/msp.js]: `ii` is not used at all at the loop, I think usually `ii++` is prettier and more habitual than `++i`

Comment (fiam) [reply src/js/msp.js]: `ii` is only used to count up to 8 iterations. If you prefer another style of looping, I can definitely change it. In the meantime, I've changed `++ii` to `ii++`.

File: src/js/msp.js (hunk 1 above)
Comment (fiam) [reply src/js/msp.js]: I've just changed all `console.log()` calls in `msp.js` to use string interpolation.

**Comment by fiam** [comment]
@McGiverGim Thanks for the review. I've just pushed the changes you requested. Please, let me know if anything else needs to be amended.

**Pull request merged**